bool ir_bumper_left
bool ir_bumper_right
bool ir_bumper_front_left
bool ir_bumper_front_right
bool ir_bumper_back


bool cliff_front_left
bool cliff_front_right
bool cliff_back_left
bool cliff_back_right

bool ultrasonic_left
bool ultrasonic_front
bool ultrasonic_right

float32 wheel_current_left
float32 wheel_current_right

float32 wheel_velocity_left
float32 wheel_velocity_right 

uint16 wheel_status_left
uint16 wheel_status_right
